src/dhara/CharFrequency.java

```diff
@@ -0,0 +1,37 @@
+package dhara;
+
+import java.util.Scanner;
+
+public class CharFrequency {
+	
+	int getCharFrequency(String string, char ch) {
+		int count = 0;
+		for(int i=0;i<string.length();i++) {
+			if(string.charAt(i)==ch) {
+				count++;
+			}
+		}
+		return count;
+	}
+	
+	public static void main(String[] args) {
+		
+		CharFrequency charFrequency=new CharFrequency();
+		System.out.println("Enter the String : ");
+		Scanner scanner=new Scanner(System.in);
+		String string=scanner.next();
+		System.out.println("Enter Charater from the String : ");
+		String temp=scanner.next();
+		while(temp.length()!=1) {
```

Comment:
Here we should give 3 chance to user as well
while(temp.length()!=1 && cnt <3){
//code
}